Ginger/GingerCoreNET/WorkSpaceLib/Solution.cs

```diff
@@ -44,7 +42,7 @@ public class Solution : RepositoryItemBase, ISolution
 
         [IsSerializedForLocalRepository]
 
-        public bool ShowIndicationkForLockedItems { get; set; }
+        public bool ShowIndicationkForLockedItems { get; set; }   // TODO: fix typo - note serialized attr
```

Comment:
have typo, 'k' in middle

Reply:
Yes - added the todo to fix it, why is it serialized?